Configurateur/front/test/container-creation/design_screen_test.dart

```diff
@@ -81,7 +81,6 @@ void main() {
 
     await tester.tap(find.text("Suivant"));
     await tester.tap(find.text("Précédent"));
-    await tester.tap(find.text("Parcourir"));
```

Comment:
Why did you delete this but it's still in the design screen page (line 680) ?

Reply:
to fix the test